controller/src/main/java/io/pravega/controller/server/rpc/grpc/v1/ControllerServiceImpl.java

```diff
@@ -101,9 +102,19 @@
     private final boolean replyWithStackTraceOnError;
 
     private final Supplier<Long> requestIdGenerator = RandomFactory.create()::nextLong;
-
+    /**
```

Comment:
Why do you have this copyright in here?

Reply:
That happened by mistake. I've removed it now.